proto/query.proto

```diff
@@ -334,6 +334,14 @@ message ExecuteOptions {
   }
 
   Consolidator consolidator = 13;
+
+  enum TransactionAccessMode {
```

Comment:
Should we be using the same name throughout - TransactionCharacteristic versus TransactionAccessMode?

Reply:
It is commonly used for `isolation level` and `access mode` so wanted to make it explicit.

```sql
START TRANSACTION
    [transaction_characteristic [, transaction_characteristic] ...]

transaction_characteristic: {
    WITH CONSISTENT SNAPSHOT
  | READ WRITE
  | READ ONLY
}
```

```sql
SET [GLOBAL | SESSION] TRANSACTION
    transaction_characteristic [, transaction_characteristic] ...

transaction_characteristic: {
    ISOLATION LEVEL level
  | access_mode
}

level: {
     REPEATABLE READ
   | READ COMMITTED
   | READ UNCOMMITTED
   | SERIALIZABLE
}

access_mode: {
     READ WRITE
   | READ ONLY
}
```

Reply:
now only the grammar uses tx_chars to match the MySQL grammar. All the output variables are `TxAccessMode`